drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-kogito-client/src/main/java/org/drools/workbench/screens/scenariosimulation/kogito/client/rightpanel/TestRunnerReportingPanelKogitoWrapperImpl.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2019 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.drools.workbench.screens.scenariosimulation.kogito.client.rightpanel;
+
+import javax.enterprise.context.Dependent;
+
+import com.google.gwt.core.client.GWT;
+import com.google.gwt.user.client.ui.IsWidget;
+import org.drools.workbench.screens.scenariosimulation.client.rightpanel.TestRunnerReportingPanelWrapper;
+import org.guvnor.common.services.shared.test.TestResultMessage;
+
+@Dependent
+public class TestRunnerReportingPanelKogitoWrapperImpl implements TestRunnerReportingPanelWrapper {
```

Comment:
Can you please verify if it is needed? I expect this should not be invoked in Kogito because we have no run feature yet

Reply:
@danielezonca Unfortunetely this is required, just because its inferface `TestRunnerReportingPanelWrapper` is injected inside `ScenarioSimulationEditorPresenter`.